hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -516,4 +547,10 @@ public int getPriority() {
       return priority;
     }
   }
+
+  private interface Deleter {
+    void apply(Table<?, DeletedBlocksTransaction> deleteTxnsTable,
+        BatchOperation batch, long txnID)
+        throws IOException;
```

Comment:
L553 and L553 can be merged into one line.

Reply:
Sure, I'll merge it.